.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I've committed the debug config for the Jest tests in case someone else has a use for it, I can also remove it if it's not customary.

Reply:
I think it's fine to commit common IDE configuration for specific tooling used in the project